packages/jumpstarter/jumpstarter/driver/base.py

```
@@ -197,15 +197,16 @@
                     finally:
                         del self.resources[uuid]
             case PresignedRequestResource(headers=headers, url=url, method=method):
+                client_timeout = aiohttp.ClientTimeout(total=timeout)
                 match method:
                     case "GET":
-                        async with aiohttp.request(method, url, headers=headers, raise_for_status=True) as resp:
+                        async with aiohttp.request(method, url, headers=headers, raise_for_status=True, timeout=client_timeout) as resp:
```

Ruff (0.8.2)

Line too long (136 > 120)

(E501)